src/index.html

```diff
@@ -1,16 +1,48 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta
+      charset="UTF-8"
+    >
+
     <meta
```

Comment:
```suggestion
    <meta charset="UTF-8">
    <meta
```

Reply:
thank you, it fixed according to checklist
I do each-attribute-in-separate-line for tags with more than 2 attributes or long class names